### PR TITLE
New Managed node group example with all the options

### PR DESCRIPTION
**Pull request by vara-bonthu**
### What does this PR do?

- New Managed node group example with all the options
- Renamed existing examples

### More

- [x] Yes, I have tested the PR using my local account setup  (Provide any test evidence report under Additional Notes)
- [ ] Yes, I have added a new example under [examples](https://github.com/aws-samples/aws-eks-accelerator-for-terraform/tree/main/examples) to support my PR
- [ ] Yes, I have created another PR for add-ons under [add-ons](https://github.com/aws-samples/ssp-eks-add-ons) repo (if applicable)
- [ ] Yes, I have updated the [docs](https://github.com/aws-samples/aws-eks-accelerator-for-terraform/tree/main/docs) for this feature
- [x] Yes, I ran `pre-commit run -a` with this PR

**Note**: Not all the PRs required examples and docs except a new pattern or add-on added.

### For Moderators
- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

**Review by kcoleman731 — APPROVED** [review]
LGTM

File: examples/windows-node-groups/README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ git clone https://github.com/aws-samples/aws-eks-accelerator-for-terraform.git
 to initialize a working directory with configuration files
 
 ```bash
-cd examples/eks-cluster-with-windows-support
```
Comment (Zvikan) [inline examples/windows-node-groups/README.md:27]: We may need to replace path on other places besides this file, see https://github.com/aws-samples/aws-eks-accelerator-for-terraform/blob/main/docs/node-groups.md?plain=1#L191

**Review by Zvikan — APPROVED** [review]
LGTM, minor change to refactor relative pathing of examples on other doc files due to the folder name changes.
Please note, after merging this, customers who will future pull changes may be confused as from their end the previous folders may stay around (due to TF state saved locally on previous folder names). 
I believe there's no way around it,  proper change log communication will do the job.

File: examples/managed-node-groups/README.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
+# EKS Cluster with Managed Node Group
+This example deploys a new EKS Cluster with a self-managed node group into a new VPC.
```
Comment (askulkarni2) [inline examples/managed-node-groups/README.md:2]: self-managed -> managed ?

File: examples/managed-node-groups/main.tf [hunk 3]
```diff
@@ -0,0 +1,468 @@
+#------------------------------------------------------------------------
```
Comment (askulkarni2) [inline examples/managed-node-groups/main.tf:1]: This file probably needs formatting.

Comment (askulkarni2) [inline examples/managed-node-groups/main.tf:1]: Also consider breaking this file into multiple files.

File: examples/windows-node-groups/README.md (hunk 1 above)
Comment (vara-bonthu) [reply examples/windows-node-groups/README.md:27]: updated

File: examples/managed-node-groups/main.tf (hunk 3 above)
Comment (vara-bonthu) [reply examples/managed-node-groups/main.tf:1]: terraform fmt updated. Comments formatted as well

Comment (vara-bonthu) [reply examples/managed-node-groups/main.tf:1]: I think the whole idea is to show all possible options to deploy managed node groups in one example hence keeping it in one file.

**Pull request merged**